### ISSUE TITLE
Bookmarks for files and directories

### ISSUE THREAD

**Issue by nealot** [alefragnani/vscode-bookmarks#142]
Hi, great extension! Would it be possible to add file and directory bookmarks? So in the activity bar panel the three categories would be:

1. Lines
2. Files
3. Directories

The behavior I'm referring to can be previewed with this extension: https://github.com/kdcro101/vscode-favorite-items. The reason I'm suggesting this feature is because I believe this extension could be the definitive method of marking files for subsequent navigation.

**Comment by alefragnani** [comment]
Hi @nealot ,

Thanks, I'm glad you like the extension!

About the file/folder bookmark, I believe it's beyond the scope of the extension. It's intention is to give an easy way to navigate between _positions_ in the text, not the file structure. It would be needed an extra effort to incorporate such a feature.

So, for this scenarios, I suggest you to use the extensions that you pointed, at least for now.

Hope this helps

**Comment by alefragnani** [comment]
Hi @nealot ,

Last week I decided to "re-focus" on some of my extensions, and I'll start to plan milestones for each of them. This "re-focus" will be based on community feedback and support.

The idea itself is interesting, but I'm still thinking its out of scope for Bookmarks extension. To me, it should be a separate extension, and in fact, I have found a few that do something like what you asked for (https://marketplace.visualstudio.com/items?itemName=kdcro101.favorites and https://marketplace.visualstudio.com/items?itemName=howardzuo.vscode-favorites). I suggest you to take a look.

Hope this helps
